specs/experimental/fault-proof/stage-one/bond-incentives.md

```diff
@@ -58,3 +64,71 @@ are located on the same level as honest claims, but situated to its right (i.e.
+
+### Bond Scaling
+
+FPM bonds are priced in the amount of gas that they are intended to cover.
+Bonds start at the very first depth of the game at a baseline of `400_000` gas.
+The `400_000` value is chosen as a deterrence amount that is approximately double the cost to respond at the top level.
+Bonds scale up to a value of `200_000_000` gas, a value chosen to cover approximately double the cost of a max-size
+Large Preimage Proposal.
+
+We use a multiplicative scaling mechanism to guarantee that the ratio between bonds remains constant.
+We determine the multiplier based on the proposed `MAX_DEPTH` of 73.
+We can use the formula `x = (200_000_000 / 400_000) ^ (1 / 73)` to determine that `x = 1.089`.
+At each depth `N`, the amount of gas charged is therefore `400_000 * (1.089 ^ N)`
+
+### Required Bond Formula
+
+Applying the [Gas Price Assumption](#gas-price-assumption) and [Bond Scaling](#bond-scaling) specifications, we have a
+`getRequiredBond` function:
+
+```python
+def get_required_bond(position):
+    assumed_gas_price = 200 gwei
+    base_gas_charged = 400_000
+    gas_charged = 400_000 * (1.089 ^ position.depth)
+    return gas_charged * assumed_gas_price
+```
+
+### DelayedWETH
+
+FPM introduces a contract `DelayedWETH` designed to hold the bonded ETH for each
+[Fault Dispute Game](./fault-dispute-game.md).
+`DelayedWETH` is an extended version of the standard `WETH` contract that introduces a delayed unwrap mechanism that
+allows the System Owner to function as a backstop in the case that a Fault Dispute Game would
+incorrectly distribute bonds.
+
+`DelayedWETH` is modified from `WETH` as follows:
+
+- `DelayedWETH` is a proxy contract owned by the System Owner.
+- `DelayedWETH` has a `startUnwrap` function that sets a timer and increments a mapping that keeps track of the
+`pending` unwrap amount for the address and the current timer for that address. Each address only has a single timer and
+the timer is always set to 7 days from the current timestamp whenever `startUnwrap` is called.
+- `DelayedWETH` has an `unwrap` function which attempts to unwrap funds from the `pending` unwrap amount but reverts if
+the timer has not yet expired. `unwrap` reverts if the MCP pause flag is enabled.
+- `DelayedWETH` has a modified `transferFrom` that allows the System Owner to transfer balances of any account.
+- `DelayedWETH` has an `ownerUnwrap` function that allows the System Owner to unwrap any amount of ETH instantly,
+regardless of the System Owner’s balance.
+
+#### Integration
+
+`DelayedWETH` is expected to be integrated into the Fault Dispute Game as follows:
+
+- When `FaultDisputeGame.move` is triggered, `DelayedWETH.wrap(msg.value)` is called.
+- When `FaultDisputeGame.resolve` is triggered, `DelayedWETH.startUnwrap(DelayedWETH.balanceOf(address(this)))` is
+called.
+- When `FaultDisputeGame.claimCredit` is triggered, `DelayedWETH.unwrap(DelayedWETH.balanceOf(address(this)))` is
```

Comment:
Is this only triggered on the first claim credit call? Seems like the entire balance of the `FaultDisputeGame` is unwrapped here, but then this is called each time someone calls `claimCredit`.